Readme.md

```diff
@@ -97,6 +97,7 @@ fastify.listen(3000, (err) => {
 {
   privacy: 'value',
   expiresIn: 300,
+  serverExpiresIn: 2592000,
```

Comment:
This is an advanced feature and i don't think we should describe it here. Also it's incompatible with `privacy: 'value'`